### PR TITLE
[LUMI-156] Drop pants

### PR DESCRIPTION
**Pull request by chainlink**
## What's changing
Completely removes pants from the repo

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -144,28 +33,4 @@ clean-docker-images:
...
-	pants --changed-since=origin/main fmt
-
-ci-tests:
-	@echo "running tests with parametrized platform: $(PARAMETRIZE)"
-	pants test lumigator/python/mzai/backend/backend/tests:backend_tests@parametrize=$(PARAMETRIZE)
-	pants test lumigator/python/mzai/sdk/tests:sdk_tests@parametrize=$(PARAMETRIZE)
-
-ci-publish-images:
-	pants package publish lumigator/python/mzai/backend:lumigator
+clean-all: clean-docker-buildcache clean-docker-containers
```
Comment (veekaybee) [inline Makefile:36]: should we consolidate clean-docker-all and clean-all at this point?

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -20,13 +16,3 @@ repos:
       - id: end-of-file-fixer
       - id: requirements-txt-fixer
         exclude: requirements_lock.txt
```
[the remark this answers is not in the document]
Comment (chainlink) [reply .pre-commit-config.yaml:18]: @macaab26 is working on re-introducing linting. It's currently broken in pants anyway

**Review by veekaybee — APPROVED** [review]
🎉

File: Makefile (hunk 1 above)
Comment (chainlink) [reply Makefile:36]: That makes sense, we might want to add a uv-clean or something too

File: .pre-commit-config.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (macaab26) [reply .pre-commit-config.yaml:18]: Yep, I will start with it tomorrow morning, I promise 😄

**Pull request merged**